core/src/bundle_stage.rs

```diff
@@ -292,12 +293,25 @@ impl BundleStage {
         bank_start: &BankStart,
         bundle_stage_leader_stats: &mut BundleStageLeaderStats,
         max_bundle_retry_duration: &Duration,
+        reserved_space: &mut u64,
     ) -> BundleStageResult<()> {
         if sanitized_bundle.transactions.is_empty() {
             return Ok(());
         }
 
         let tx_costs = qos_service.compute_transaction_costs(sanitized_bundle.transactions.iter());
+        let total_cost = tx_costs.iter().fold(0, |acc, cost| acc + cost.sum());
```

Comment:
should be able to do iter().sum() i think?

Reply:
`error[E0277]: the trait bound `u64: Sum<&TransactionCost>` is not satisfied
    --> core/src/bundle_stage.rs:306:42
     |
306  |         let total_cost = tx_costs.iter().sum();
     |                                          ^^^ the trait `Sum<&TransactionCost>` is not implemented for `u64`
`

Reply:
prob have to give some type hints in the sum stuff